src/Ng/Scripts/Catalog2LuceneV3-Reg2.cmd

```diff
@@ -0,0 +1,26 @@
+@echo OFF
+
+cd Ng
+
+:Top
+echo "Starting job - #{Jobs.catalog2lucenev3reg2.Title}"
+
+title #{Jobs.catalog2lucenev3reg2.Title}
+
+start /w ng.exe catalog2lucene ^
+    -source #{Jobs.common.v3.Source} ^
```

Comment:
Why is this `Jobs.common.v3.Source` but the source for secondary above is `Jobs.catalog2lucenev3reg2secondary.Source`?